### PR TITLE
improve unit testing and the code coverage

### PR DESCRIPTION
**Pull request by peter279k**
# Change log
- complete the unit testing.
- improve the code coverage for this unit testing.

[travis-ci status](https://travis-ci.org/peter279k/spdx-licenses)

[code coverage report ](https://codecov.io/gh/peter279k/spdx-licenses)(provided by codecov)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 language: php
 
+sudo: false
```
Comment (alcohol) [inline .travis.yml]: This is default, please remove.

Comment (peter279k) [inline .travis.yml]: Hi @alcohol , than you for your comment.
I will fix it.

**Comment by alcohol** [comment]
Thanks.

**Comment by peter279k** [comment]
Hi @alcohol , I think the ```.travis.yml``` is not merged well...
When I check ```sudo``` config, it still set this configuration. It need to be modified it manually.
Thanks.

**Comment by alcohol** [comment]
You are right. It seems to be the case that it was already in the file. Just at a different spot. See https://github.com/peter279k/spdx-licenses/blob/ff0fde2a8445e37c393de753e7ba9103ba776239/.travis.yml#L3-L12

Fixed in 7fc8d83

**Pull request merged**